lang/python/django/Makefile

```
@@ -21,46 +21,51 @@ PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=LICENSE LICENSE.python
 PKG_CPE_ID:=cpe:/a:djangoproject:django
 
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-django-$(PKG_VERSION)
+
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
 
-define Package/django
-    SUBMENU:=Python
-    SECTION:=lang
-    CATEGORY:=Languages
-    TITLE:=The web framework for perfectionists with deadlines.
-    URL:=https://www.djangoproject.com/
-    DEPENDS:=+python +python-pytz
+define Package/django/Default
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=The web framework for perfectionists with deadlines.
+  URL:=https://www.djangoproject.com/
 endef
 
-define Package/django/description
-    The web framework for perfectionists with deadlines.
+define Package/python-django
+$(call Package/django/Default)
+  DEPENDS:= \
+	+PACKAGE_python-django:python \
+	+PACKAGE_python-django:python-pytz
+  VARIANT:=python
 endef
 
-define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
+
+define Package/python-django/description
+    The web framework for perfectionists with deadlines.
 endef
 
-define Build/InstallDev
-	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) \
-	    $(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)/* \
-	    $(1)$(PYTHON_PKG_DIR)
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/django-admin.py \
-	    $(1)/usr/bin
+define Package/python3-django
+$(call Package/django/Default)
+  DEPENDS:= \
+	+PACKAGE_python3-django:python3 \
+	+PACKAGE_python3-django:python3-pytz
+  VARIANT:=python
```

```suggestion
  VARIANT:=python3
```
This is probably causing the build failure.